snapshot_test.go

```diff
@@ -1,25 +1,6 @@
 package frostdb
 
-import (
-	"context"
-	"fmt"
-	"math"
-	"os"
-	"strconv"
-	"testing"
-	"time"
-
-	"github.com/apache/arrow/go/v14/arrow"
-	"github.com/apache/arrow/go/v14/arrow/array"
-	"github.com/apache/arrow/go/v14/arrow/memory"
-	"github.com/stretchr/testify/require"
-	"golang.org/x/sync/errgroup"
-
-	"github.com/polarsignals/frostdb/dynparquet"
-	"github.com/polarsignals/frostdb/query"
-	"github.com/polarsignals/frostdb/query/logicalplan"
-)
-
+/*
```

Comment:
This is likely WIP, but adding a comment here to make sure we enable these tests before merging.

Reply:
Whoops those should be enabled again. Thanks!